### PR TITLE
Add tests for ModalContext

### LINKED ISSUE #209: add tests for ModalContext

### PR DESCRIPTION
**Pull request by mhevyk**
image

image

### CODE REVIEW

File: code/src/context/modal/ModalContext.test.tsx [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+      jest.restoreAllMocks();
+    });
+
+    test("throws an error when context in not within a provider", () => {
+      jest.spyOn(console, "error").mockImplementation(() => {});
+
+      const errorMessage =
+        "useModalContext must be used within a ModalProvider";
+
+      expect(() => renderHook(() => useModalContext())).toThrow(errorMessage);
```
Comment (heorhiiss) [inline code/src/context/modal/ModalContext.test.tsx]: Maybe move `() => renderHook(() => useModalContext())` to separate variable?

Comment (mhevyk) [reply code/src/context/modal/ModalContext.test.tsx]: Moved

**Comment by heshvab** [comment]
I see that when user closes modal, it instantly disappears, without animation. Shouldn't there be some closing animation? Since this pr is for tests, maybe would be good to create some issue for that (if that is not desired behaviour).

**Review by D1nviz — APPROVED** [review]

**Comment by mhevyk** [comment]
> I see that when user closes modal, it instantly disappears, without animation. Shouldn't there be some closing animation? Since this pr is for tests, maybe would be good to create some issue for that (if that is not desired behaviour).

It is not nessesary now. Let's discuss it with the team. As far as I know, @D1nviz struggled with this functionality. If we really need this, we can create separate issue for it

**Review by heshvab — APPROVED** [review]

**Review by dmtrth25 — APPROVED** [review]

**Pull request merged**